config/sys.config

```diff
@@ -104,6 +104,7 @@
             prometheus_vm_statistics_collector,
             prometheus_vm_system_info_collector
         ]},
+        {instrumenters, []},
```

Comment:
I am curious to know what does this exactly do? Does that remove a bunch of instruments that come by default?

Reply:
If the list of `instrumenters` is not configured, prometheus library loads all modules and checks which of them have `behaviour` attribute set to `prometheus_instrumenter`. The code loading takes a lot of time. At the moment there is only one such module, `prometheus_test_instrumenter`.

https://github.com/deadtrickster/prometheus.erl/blob/master/src/prometheus_instrumenter.erl#L43
https://github.com/deadtrickster/prometheus.erl/blob/master/src/prometheus_misc.erl